conf/logback.xml

```diff
@@ -17,7 +17,7 @@ https://www.playframework.com/documentation/2.4.x/SettingsLogger#Using-a-configu
         </rollingPolicy>
 
         <encoder>
-            <pattern>%date [%.-30thread] %logger[%file:%L] %highlight(%level: %msg%n%xException{3})</pattern>
+            <pattern>%date %logger[%file:%L] %level: %msg%n%xException</pattern>
```

Comment:
the logger was putting lots of strange characters in which are supposed to do colouring, but it wasn't working.  Also removed the thread id (it's long but is it needed?) and let it log the whole stack traces.